### PR TITLE
Feature/zebpay

### PR DESCRIPTION
**Pull request by cySheng**
- Implemented Pairs, Tickers and updated README for Zebpay exchange
- Issue #357 
- [X] I have added Specs
- [X] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [X] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

**Comment by tmlee** [comment]
@cySheng if you can try to keep this branch separate from the `coinroom` changes

File: lib/cryptoexchange/services/market.rb [hunk 1]
```diff
@@ -32,7 +32,12 @@ def fetch(endpoint)
 
       def fetch_using_post(endpoint, params, submit_as_form = false)
         LruTtlCache.ticker_cache.getset(endpoint) do
-          response = http_post(endpoint, params)
+          response = 
+            if submit_as_form
```
Comment (tmlee) [inline lib/cryptoexchange/services/market.rb]: this change is unrelated to zebpay implementation? @cySheng

**Comment by cySheng** [comment]
Apologies, you are right.
